public_html/wp-content/plugins/wc-post-types/inc/deprecated.php

```diff
@@ -0,0 +1,616 @@
+<?php
+/**
+ * Deprecated functionality.
+ *
+ * This is still loaded on all WordCamp sites, but is considered unsupported. Instead of using shortcodes to
+ * display content, the corresponding blocks should be used.
+ */
+
+class WordCamp_Post_Types_Plugin_Deprecated {
+	/**
+	 * Set up functionality.
+	 */
+	public function __construct() {
+		add_shortcode( 'speakers', array( $this, 'shortcode_speakers' ) );
+		add_shortcode( 'sessions', array( $this, 'shortcode_sessions' ) );
+		add_shortcode( 'sponsors', array( $this, 'shortcode_sponsors' ) );
+		add_shortcode( 'organizers', array( $this, 'shortcode_organizers' ) );
+	}
+
+	/**
+	 * Convert a string representation of a boolean to an actual boolean
+	 *
+	 * @param string|bool $value
+	 *
+	 * @return bool
+	 */
+	public function str_to_bool( $value ) {
```

Comment:
I wonder if this function is really needed anymore. Core has `wp_validate_boolean` now, which is very similar, except that it treats more things as truthy.

- [ ] `[MIGHT]` It may not be worth the time to test for behavior changes, but if it's quick, it would be nice to eliminate this redundant custom function in favor of Core's `wp_validate_boolean`.

Reply:
`wp_validate_boolean` seems more in line with expected behavior too, so I've switched that over.